view-user/src/components/common/Loading/Loading.module.css

```
@@ -0,0 +1,58 @@
+/* Loading */
+.container {
+  background-color: white;
+  margin: 9.375vw;
+  border-radius: 10vw;
+}
```

```suggestion
.container {
  background-color: white;
  margin: 9.375vw;
  border-radius: 10vw;
  filter: drop-shadow(1.25vw 1.25vw 0px #ff3342);
}
```
figmaを見たら分かりますがdrop shadowがあります